torch/_subclasses/fake_tensor.py

```diff
@@ -2120,6 +2120,7 @@ def _dispatch_impl(
             and len(flat_arg_fake_tensors) != 0
             and not has_symbolic_sizes
             and not avoiding_device_init
+            and func is not aten._nested_tensor_from_tensor_list.default
```

Comment:
what's going on here?

Reply:
This one is just annoying. There is a fake kernel associated with this op which just raises exception. However, that kernel is called later on. There is a fast path in fake_tensor where if all inputs are constant, then we do something else. And for this op, going through that fast patch causes issues.